### DISCUSSION TITLE
School Emblems in CV

### DISCUSSION THREAD

**Discussion by LiuYing-1** [alshedivat/al-folio#2148] [Q&A] [answered]
Hi guys! 

Is it possible to insert the school emblems in the cv part, for different education? I have checked a bit on the code, and it seems that the values are stored in the .yml file, and that might be impossible to achieve. If some of you can help me a bit, thanks a lot in advance!

Best,
Ying

**Comment by george-gca** [comment, marked as answer]
It is possible, but you'll have to create the code for it, since it is not implemented.

Besides the yml file, you'll also need to change something in [_layouts/cv.liquid](https://github.com/alshedivat/al-folio/blob/master/_layouts/cv.liquid) and some files in [_includes/cv/](https://github.com/alshedivat/al-folio/tree/master/_includes/cv).

**Reply by LiuYing-1** [reply]
Hi George!

Thanks for your reply!

However, I thought I am currently using the older version which .liquid files are not involved. Therefore, I am thinking that what is the correct way to 'upgrade' my project since I have done a lot of modifications before and I do not want to lose them.

Best,
Ying

**Reply by george-gca** [reply]
See [our installation instructions](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#upgrading-from-a-previous-version) on this.

**Reply by LiuYing-1** [reply]
This helps me a lot, thanks - Ying
